soci/soci_convert.go

```diff
@@ -146,6 +146,12 @@ func (b *IndexBuilder) Convert(ctx context.Context, img images.Image, opts ...Co
 		return nil, err
 	}
 
+	// Fix artifacts DB references
```

Comment:
nit: isn't it better to call this as `update artifacts DB references`. Using fix gives the impression that something is broken.